pkgs/applications/audio/osmid/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, alsaLib
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  name = "osmid";
```

Comment:
```suggestion
  pname = "osmid";
```

Reply:
Done

Reply:
Thanks @alexarice... somehow my eyes glazed over that one :+1: